packages/config-array/src/config-array.js

```diff
@@ -551,12 +545,19 @@ const dataCache = new WeakMap();
  * those config objects.
  */
 export class ConfigArray extends Array {
+	/** The namespaced path of the config file directory. */
+	#namespacedBasePath;
+
+	/** Path-handling implementations */
+	#path;
+
```

Comment:
Can we add types to these properties?

Reply:
Done in 98b9a76bbf1dc74afd8dca5b4548aa8811e7213b.

Reply:
Done in ab61572782b18afdf0b3c3bb1999b641f3d94f2c.